README.md

```diff
@@ -71,7 +76,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+  def friends.each do |befriend|
```

Comment:
missed this first time around, why do you have the `def` here? We're not defining a method, but rather calling the `each` method on the friends array.